### PR TITLE
Security Pen and Penlight improvement

### PR DESCRIPTION
**Pull request by DrDiasyl**
## About The Pull Request
Improves penlight sprite and code.

A new type of pen has been introduced - the Security Pen. This pen is red and can create a holograph around a person, similar to a medical penlight. The holograph prompts the person to surrender for 30 seconds, and the holograph itself is visible for 5. There is a 30-second cooldown between usage.

Holosign itself does nothing except be visual, just like medical holosign **(DO NOT MISTAKE HOLOSIGNS FOR HOLOBARRIERS)**

https://github.com/tgstation/tgstation/assets/42353186/1ee5f794-7218-4e52-b04f-3ebb50bd224a

Now every Security member spawns with a Security Pen in their PDA.

Now you can print Security Pens at Security Techfab, and Penlights at Medical Techfab.
## Why It's Good For The Game
The only way to prompt someone to perform the surrender emote is by holding them up with a gun. However, this mechanic can be quite unreliable, even after updates, especially if the person is moving. As a result, many players are unaware of how to prompt the surrender emote or even about its existence, so they don't know they can trigger it themselves by typing "*surrender".
This enables non-lethal arrests without stun batons or other tools.

Sprites are a significant improvement as the previous penlight sprites were outdated.
## Changelog
:cl:
qol: There is now a more convenient way to prompt surrender to emote - the Security Pen. Each officer is equipped with one in their PDA. Simply click someone with it to prompt a 30-second surrender. They are printable at Security Techfab as well.
qol: Penlights now are printable at Medical Techfab.
image: Penlights got a new cleaner sprite to replace its ancient one
code: The code for Penlight's holographic sign has been improved.
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
30 seconds might be a bit much for that overlay remaining there, I'd probably make it 5 or 10 seconds instead.

**Comment by DrDiasyl** [comment]
> 30 seconds might be a bit much for that overlay remaining there, I'd probably make it 5 or 10 seconds instead.

The holosign seen on a person only lasts 4-5 seconds. The prompt is remaining for 30 seconds, if you are talking about it.

**Comment by GoldenAlpharex** [comment]
> > 30 seconds might be a bit much for that overlay remaining there, I'd probably make it 5 or 10 seconds instead.
> 
> The holosign seen on a person only lasts 4-5 seconds. The prompt is remaining for 30 seconds, if you are talking about it.

Perfect, I misunderstood the PR body, then.

I like this.

**Comment by DrDiasyl** [comment]
`[17:56:31] Runtime in stack_trace.dm,4: We somehow ended up with a qdeleted or null object in SSwardrobe's stock. Something's weird, likely to do with reinsertion. Typepath of /obj/item/flashlight/pen (code/controllers/subsystem/wardrobe.dm:262)`

No idea how to fix this

**Comment by DrDiasyl** [comment]
Those runtimes in the Monkey Business really seem to have nothing to do with this PR.

**Comment by JupiterJaeden** [comment]
Top tier PR

**Review by Wallemations — APPROVED** [review]
Nice sprite and especially clean hologram animation

**Comment by deathrobotpunch** [comment]
> Now those new runtimes in the Monkey Business seem to have nothing to do with this PR.

yeah it’s due to a runtime caused by a monkey attacking a parrot

**Comment by Wallemations** [comment]
Since this touches code more than a little bit I won't merge it myself but know the sprites are good to go for any code maints that _do_ wanna merge this

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -400,3 +400,44 @@
 	. = ..()
 	icon_state = "[initial(icon_state)][HAS_TRAIT(src, TRAIT_TRANSFORM_ACTIVE) ? "_out" : null]"
 	inhand_icon_state = initial(inhand_icon_state) //since transforming component switches the icon.
+
+//The Security holopen
+/obj/item/pen/red/security
+	name = "security pen"
+	desc = "This is a red ink pen exclusively provided to members of the Security Department. Its opposite end features a built-in holographic projector designed for issuing arrest prompts to individuals."
+	icon_state = "pen_sec"
+	var/holo_cooldown = 0
```
Comment (Jacquerel) [inline code/modules/paperwork/pen.dm:409]: Can we use the `COOLDOWN_DECLARE` / `COOLDOWN_START` / `COOLDOWN_FINISHED` macros for this?
Would be nice as a slightly code improvement on the medical penlight too but you didn't write that one.

File: code/modules/paperwork/pen.dm [hunk 2]
```diff
@@ -400,3 +400,44 @@
...
+
+/obj/item/pen/red/security/examine(mob/user)
+	. = ..()
+	. += span_notice("To initiate the surrender prompt, simply click on an individual within your proximity.")
+
+//Code from the medical penlight
+/obj/item/pen/red/security/afterattack(atom/target, mob/living/user, proximity)
+	. = ..()
+	if(holo_cooldown > world.time)
+		to_chat(user, span_warning("[src] is not ready yet!"))
```
Comment (Jacquerel) [inline code/modules/paperwork/pen.dm:419]: Ephemeral enough information to be a balloon alert.

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Couple of small suggestions & this PR is in conflict

**Pull request merged**